[2024-06-04T22:54:22.948+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-04T22:54:22.965+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingestion_data.transformacion_de_datos manual__2024-06-04T22:54:21.290719+00:00 [queued]>
[2024-06-04T22:54:22.971+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingestion_data.transformacion_de_datos manual__2024-06-04T22:54:21.290719+00:00 [queued]>
[2024-06-04T22:54:22.972+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-04T22:54:22.981+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformacion_de_datos> on 2024-06-04 22:54:21.290719+00:00
[2024-06-04T22:54:22.988+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=65) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-04T22:54:22.989+0000] {standard_task_runner.py:63} INFO - Started process 67 to run task
[2024-06-04T22:54:22.989+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ingestion_data', 'transformacion_de_datos', 'manual__2024-06-04T22:54:21.290719+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/entregable_dag.py', '--cfg-path', '/tmp/tmpvr_rfdjp']
[2024-06-04T22:54:22.990+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask transformacion_de_datos
[2024-06-04T22:54:23.033+0000] {task_command.py:426} INFO - Running <TaskInstance: ingestion_data.transformacion_de_datos manual__2024-06-04T22:54:21.290719+00:00 [running]> on host fd0d5636e48d
[2024-06-04T22:54:23.095+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Guillermo Hern√°ndez' AIRFLOW_CTX_DAG_ID='ingestion_data' AIRFLOW_CTX_TASK_ID='transformacion_de_datos' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T22:54:21.290719+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-04T22:54:21.290719+00:00'
[2024-06-04T22:54:23.096+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-04T22:54:24.145+0000] {python.py:237} INFO - Done. Returned value was:      Latitude  Longitude  ...  Wind_Speed_10M              Creation_Date
0       52.52  13.419998  ...             6.8 2024-06-04 22:54:24.140946
1       52.52  13.419998  ...             5.4 2024-06-04 22:54:24.140946
2       52.52  13.419998  ...             6.5 2024-06-04 22:54:24.140946
3       52.52  13.419998  ...             6.5 2024-06-04 22:54:24.140946
4       52.52  13.419998  ...             6.3 2024-06-04 22:54:24.140946
..        ...        ...  ...             ...                        ...
163     52.52  13.419998  ...             6.2 2024-06-04 22:54:24.140946
164     52.52  13.419998  ...             4.7 2024-06-04 22:54:24.140946
165     52.52  13.419998  ...             3.4 2024-06-04 22:54:24.140946
166     52.52  13.419998  ...             3.3 2024-06-04 22:54:24.140946
167     52.52  13.419998  ...             3.8 2024-06-04 22:54:24.140946

[168 rows x 16 columns]
[2024-06-04T22:54:24.175+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-04T22:54:24.214+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ingestion_data, task_id=transformacion_de_datos, run_id=manual__2024-06-04T22:54:21.290719+00:00, execution_date=20240604T225421, start_date=20240604T225422, end_date=20240604T225424
[2024-06-04T22:54:24.248+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-04T22:54:24.282+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-04T22:54:24.283+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
